src/core/infrastructure/repository/manager.ts

```diff
@@ -0,0 +1,7 @@
+export const getGestionnaire = (addresse: string): string | null => {
+  if (addresse && addresse.includes('Paris')) {
+    return 'Paris';
```

Comment:
I have doubts about this method.
Ex. If I have an address like '5 Route de Paris, Amiens, France', it will return 'true' but the manager should not be these for one in Paris. (But maybe I misunderstood the purpose of the MR)

Reply:
yes we could have false positivie. But it's a first draft so I think we can go with this